pyproject.toml

```diff
@@ -38,7 +38,7 @@ Documentation = "https://typer.tiangolo.com/"
 [tool.flit.metadata.requires-extra]
 test = [
     "shellingham >=1.3.0,<2.0.0",
-    "pytest >=4.4.0,<5.4.0",
+    "pytest >=6.2.5",
```

Comment:
Not sure if I should bump it on this PR, but it was necessary to test it under Python 3.10